acme/client_test.go

```diff
@@ -301,7 +305,7 @@ func writeJSONResponse(w http.ResponseWriter, body interface{}) {
 }
 
 // stubValidate is like validate, except it does nothing.
-func stubValidate(j *jws, domain, uri string, chlng challenge) error {
+func stubValidate(_ *jws, _, _ string, _ challenge) error {
```

Comment:
tbh, the variables before I felt were more readable. This one is really hard to understand.

Reply:
the parameters names are define in `type validateFunc func(j *jws, domain, uri string, chlng challenge) error` 

https://github.com/xenolf/lego/blob/master/acme/client.go#L55